internal/business/rules/exclude_subgraph_errors/exclude_subgraph_errors.go

```diff
@@ -0,0 +1,39 @@
+package exclude_subgraph_errors // nolint:revive
+
+type Config struct {
+	Enabled bool `conf:"default:true" yaml:"enabled"`
+}
+
+type ExcludeSubgraphErrors struct {
+	enabled bool
+}
+
+func NewExcludeSubgraphErrors(cfg Config) *ExcludeSubgraphErrors {
+	return &ExcludeSubgraphErrors{
+		enabled: cfg.Enabled,
+	}
+}
+
+func (a *ExcludeSubgraphErrors) ProcessBody(payload map[string]interface{}) map[string]interface{} {
+	redactedErrorArray := []map[string]interface{}{
```

Comment:
this can be a global var

Reply:
do we want to return this as one error, even if many errors got returned? 
I imagine not much value from seeing `Error(s) redacted` many times, but want to ensure it is a conscious decision

Reply:
yea its indeed a consious decision to return a single array with 1 error in it